.github/workflows/workflow.yaml

```
@@ -11,6 +11,14 @@ on:
       - "website/**"
     branches: [master]
 jobs:
+  website_script_unit_test:
+    runs-on: ubuntu-latest
+    name: "Test scripts"
+    steps:
+      - uses: actions/checkout@v2
```

Just curious, is there a need to stay with v2?  I can't recall if v2 is using the Node 12 runtime which is deprecated later this year.

Alternatively we could do a follow-up issue/PR bumping all `actions/checkout`

```suggestion
      - uses: actions/checkout@v3
```

---

updated with sha